lib/GADS/API.pm

```diff
@@ -692,7 +692,7 @@ sub _post_add_user_account
     my $update_user;
     if ($id)
     {
-        $update_user = schema->resultset('User')->active->find($id)
+        $update_user = schema->resultset('User')->find($id)
```

Comment:
I'm not sure about the above change - is there a reason for removing the search for only in active (non-deleted, accepted) users?

Reply:
Hi Andy,

During testing, only searching in active users prevented account requests from being accepted. The API endpoint will only be used by the wizard for creating new users, or accepting account requests. Edit user is done via the GADS.pm endpoint. Thats why i had to remove the active check.

```
sub active
{   my ($self, %search) = @_;

    $self->search({
        account_request => 0,
        'me.deleted'    => undef,
        %search,
    });
}
```